src/components/MultiHopTrade/components/TradeConfirm/hooks/useStreamingProgress.tsx

```diff
@@ -1,29 +1,21 @@
-import type { TradeQuoteStep } from '@shapeshiftoss/swapper'
+import type { Swap } from '@shapeshiftoss/swapper'
 import { SwapperName } from '@shapeshiftoss/swapper'
 
 import { useChainflipStreamingProgress } from './useChainflipStreamingProgress'
 import { useThorStreamingProgress } from './useThorStreamingProgress'
 
-import {
-  selectActiveQuote,
-  selectActiveSwapperName,
-} from '@/state/slices/tradeQuoteSlice/selectors'
-import { useAppSelector } from '@/state/store'
-
 type UseStreamingProgressProps = {
-  hopIndex: number
-  tradeQuoteStep: TradeQuoteStep
+  swap: Swap | undefined
```

Comment:
Not multi-hop anymore but since streaming swaps are not multi-hops either way, meh, might as well!